templates/test/ci/cluster-template-prow-ipv6.yaml

```diff
@@ -263,6 +263,14 @@ spec:
         /etc/resolv.conf
       - systemctl restart systemd-resolved
 ---
+apiVersion: kubeproxy.config.k8s.io/v1alpha1
+kind: KubeProxyConfiguration
```

Comment:
this is going to get created on the management cluster is that the intention?

Reply:
:(